Documentation/getting-started-guide.md

```diff
@@ -49,87 +49,27 @@ $ ldd hello
 	not a dynamic executable
 ```
 
-## Create the image manifest
-
-Edit: manifest.json
-
-```json
-{
-    "acKind": "ImageManifest",
-    "acVersion": "0.7.0",
-    "name": "example.com/hello",
-    "labels": [
-        {
-            "name": "version",
-            "value": "0.0.1"
-        },
-        {
-            "name": "arch",
-            "value": "amd64"
-        },
-        {
-            "name": "os",
-            "value": "linux"
-        }
-    ],
-    "app": {
-        "user": "root",
-        "group": "root",
-        "exec": [
-            "/bin/hello"
-        ],
-        "ports": [
-            {
-                "name": "www",
-                "protocol": "tcp",
-                "port": 5000
-            }
-        ]
-    },
-    "annotations": [
-        {
-            "name": "authors",
-            "value": "Kelsey Hightower <kelsey.hightower@gmail.com>"
-        }
-    ]
-}
-```
-
-### Validate the image manifest
-
-To validate the manifest, we can use `actool`, which is currently provided in [releases in the App Container repository](https://github.com/appc/spec/releases).
-
-```
-$ actool --debug validate manifest.json
-manifest.json: valid ImageManifest
-```
-
-## Create the layout and the rootfs
+## Create the image
 
-```
-$ mkdir hello-layout/
-$ mkdir hello-layout/rootfs
-$ mkdir hello-layout/rootfs/bin
-```
+To create the image, we can use `acbuild`, which can be downloaded via one of
```

Comment:
Oh, it was wrapping at like 28 a little ways up in the file so I was just following that :p.